### PR TITLE
finished mkl-6 Design and Develope login template

### PR DESCRIPTION
**Pull request by Rantama**
-completed login template

### CODE REVIEW

**Review by tarunktk — COMMENTED** [review]
-login page works
-responsive

File: client/app/account/login/login.html [hunk 1]
```diff
@@ -1,47 +1,61 @@
...
+                                </p>
+                                <p class="help-block" ng-show="form.email.$error.email && vm.submitted">
+                                    Please enter a valid email.
+                                </p>
+
+                                <p class="help-block">{{ vm.errors.other }}</p>
+                            </div>
+
+                            <div>
+                                <button class="md-raised" class="btn btn-inverse btn-lg btn-login" type="submit">
```
Comment (lchapoy) [inline client/app/account/login/login.html]: The button does not have material design

File: client/app/account/login/login.html [hunk 2]
```diff
@@ -1,47 +1,61 @@
...
-          <button class="btn btn-inverse btn-lg btn-login" type="submit">
-            Login
-          </button>
-          <a class="btn btn-default btn-lg btn-register" ui-sref="signup">
-            Register
-          </a>
-        </div>
-
-      </form>
+<div layout-margin flex layout="row" layout-align="center">
```
Comment (lchapoy) [inline client/app/account/login/login.html]: The login shouldn't  take all the length, just the required length

**Review by lchapoy — CHANGES_REQUESTED** [review]
Design changes

**Review by tarunktk — COMMENTED** [review]
side panel removed.
used angular material
responsive

**Review by lchapoy — APPROVED** [review]

**Pull request merged**